src/views/research/research.jsx

```diff
@@ -0,0 +1,185 @@
+                            Some Reflections on Designing Construction Kits for Kids</a>. Proceedings of Interaction Design and{' '}
+                            Children conference, Boulder, CO.</li>
+                        <li>Maloney, J., Burd, L., Kafai, Y., Rusk, N., Silverman, B., and Resnick, M. (2004).{' '}
+                            <a class="external" href="https://llk.media.mit.edu/papers/ScratchSneakPreview.pdf">Scratch: A Sneak Preview</a>.{' '}
+                            Second International Conference on Creating, Connecting, and Collaborating through Computing. Kyoto, Japan, pp. 104-109.</li>
+                        </ul>
+                        <h3><FormattedMessage id='research.grants' /></h3>
+                        <ul>
+                        <li>Resnick, M., Rusk, N., &amp; Dasgupta, S. (2013). <a class="external" href="http://nsf.gov/awardsearch/showAward?AWD_ID=1417663">
+                            New Pathways into Data Science: Extending the Scratch Programming Language to Enable Youth to Analyze{' '}
+                            and Visualize Their Own Learning</a> (NSF DRL-1417663).</li>
+                        <li>Resnick, M., Ito, M., Gasser, U., &amp; Rusk, N. (2013). <a class="external" href="http://www.nsf.gov/awardsearch/showAward?AWD_ID=1348876">
+                            Coding for All: Interest-Driven Trajectories to Computational Fluency</a> (NSF IIS-1348876).</li>
+                        <li>Bers, M., &amp; Resnick, M. (2011). <a class="external" href="http://www.nsf.gov/awardsearch/showAward?AWD_ID=1118664">
+                            ScratchJr: Computer Programming in Early Childhood Education as a Pathway to Academic Readiness and Success</a> (NSF DRL-1118682).</li>
+                        <li>Resnick, M., Kafai, Y., Benkler, Y., Rusk, N., Maloney, J., &amp; Monroy-Hernández, A. (2010).{' '}
+                            <a class="external" href="http://www.nsf.gov/awardsearch/showAward?AWD_ID=1027848">
+                                Preparing the Next Generation of Computational Thinkers: Transforming Learning and Education Through Cooperation in Decentralized Networks
+                            </a> (NSF DRL-1027848).</li>
+                        <li>Resnick, M., &amp; Brennan, K. (2010). <a class="external" href="http://www.nsf.gov/awardsearch/showAward?AWD_ID=1019396">
+                            ScratchEd: Working with Teachers to Develop Design-Based Approaches to the Cultivation of Computational Thinking</a> (NSF DRL-1019396)</li>
+                        <li>Resnick, M., Maloney, J., Rusk, N., &amp;  Monroy-Hernández, A. (2010).{' '}
+                            <a class="external" href="http://www.nsf.gov/awardsearch/showAward?AWD_ID=1002713">
+                                Scratch 2.0: Cultivating Creativity and Collaboration in the Cloud</a> (NSF IIS-1002713).</a></li>
```

Comment:
I think the problem is that you have two </a>'s

Reply:
This is making it so that the </li> (in red) and the </ul> (in red) are messed up

Reply:
I’m pretty sure that that’s besides the point - `npm test` does not throw any errors up until `webpack --bail`

Reply:
The travis build shows that error thouhg.

Reply:
I've actually fixed that locally FYI - I'm not pushing yet because I can't tell whether it's my own code or a problem with the repo that's causing everything to fail

Reply:
Oh okay.

Reply:
@Kenny2github it may be worth pushing that fix for the `</a>` to see if it passes on travis. If it does, then it's definitely an environment issue that's causing the build to fail for you.

Reply:
eh, true